lib/ansible/cli/galaxy.py

```diff
@@ -1298,7 +1298,7 @@ def execute_install(self, artifacts_manager=None):
         collection_requirements = []
         role_requirements = []
         if context.CLIARGS['type'] == 'collection':
-            collection_path = GalaxyCLI._resolve_path(context.CLIARGS['collections_path'])
+            collection_path = GalaxyCLI._resolve_path(context.CLIARGS['collections_path'] or AnsibleCollectionConfig.collection_paths)
```

Comment:
IIRC, not installing to the python sys paths was intended, so I think this could stay as it was. `context.CLIARGS['collections_path']` will either be `-p` or the first explicitly configured collections path.

Reply:
Got it, I overlooked that `install_parser` uses a different default for `collections_path` than `verify` and `list`

Reply:
Resolved